source/jormungandr/jormungandr/scenarios/helper_classes/transfer.py

```diff
@@ -255,36 +288,48 @@ def _get_transfer_result(self, section, origin, destination):
             sub_request_id,
         )
         if direct_path and direct_path.journeys:
-            return TransferResult(direct_path, origin, destination)
-        return None
-
-    def _do_access_point_transfer(self, section, prev_section_mode, next_section_mode):
-        access_points = self.get_underlying_access_points(section, prev_section_mode, next_section_mode)
+            return (
```

Comment:
We just need to be cautious/aware of the huge amount of string that could be generated but this thus impacting the performance cache/redis. On top of that, serialize/deserialize could be time-consuming too